docs/contributing/questions/asking-questions.md

```diff
@@ -10,15 +10,15 @@ In case you're facing an issue, consider the following steps before posting a qu
 
 - **Suspect a bug?** Refer to our [Reporting Bugs] guide to determine if you've encountered a Detox bug and how to report it.
 - **Search** for existing answers that may have addressed your question already. [Below](#where-to-ask), we've listed the right platforms to ask (and search) questions.
-- Review our **documentation** thoroughly for any related information. Use the search bar on the top bar to streamline your search.
+- Review our **documentation** thoroughly for any related information. Use <a href="#" onClick={() => document.querySelector('.DocSearch-Button').click()}>the search bar</a> in the top right corner to streamline your search.
```

Comment:
1. I thought it's a good idea to help to find dynamically that search bar.
2. Top right corner is more specific.